plugins/board-resources/src/components/CardActions.svelte

```diff
@@ -0,0 +1,80 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+// Copyright © 2021 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import type { Card } from '@anticrm/board'
+  import { Button, IconAttachment, Label } from '@anticrm/ui'
+  import board from '../plugin'
+
+  export let value: Card
+</script>
+
+{#if value}
+  <div class="root">
+    <div class="section">
+      <Label label={board.string.AddToCard} />
+      <Button icon={board.icon.Card} label={board.string.Members} kind="no-border" justify={'left'} width={'100%'} />
```

Comment:
Could we have CardActivity interface with some action, component and allow to contribute them using model, instead of having a lot of predefined buttons here? It will be more extensible.

Reply:
Updated